ci-operator/step-registry/agent-qe/baremetal/install/ove/disconnected/mirror-operators/agent-qe-baremetal-install-ove-disconnected-mirror-operators-commands.sh

```
@@ -0,0 +1,479 @@
+# The script exit 0 if fail to mirror images. this allows the other test can be executed continuously
+# The script exit 0 if fail to create catalogsource. this allows  the other test can be executed continuously
+set -ux
+# use it as a bool
+marketplace=0
+
+
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)